### PR TITLE
Adding vtctl RPC interface to vtcombo, with unit test.

### PR DESCRIPTION
**Pull request by alainjobart**
@sougou @michael-berlin
Now for instance 'Ping' in the exported vtctld UI works, or remote vtctl commands using vtctlclient to vtcombo also work.

### CODE REVIEW

File: go/cmd/vtcombo/tablet_map.go [hunk 1]
```diff
@@ -161,11 +168,21 @@ func initTabletMap(ts topo.Server, topology string, mysqld mysqlctl.MysqlDaemon,
 		}
 	}
 
-	// Register the tablet dialer
+	// Register the tablet dialer for tablet server
 	tabletconn.RegisterDialer("internal", dialer)
 	*tabletconn.TabletProtocol = "internal"
+
+	// Register the tablet dialer for tablet manager
```
Comment (michael-berlin) [inline go/cmd/vtcombo/tablet_map.go]: nit: You could have written here "tablet manager dialer" instead to distinguish between the two?

File: go/cmd/vtcombo/tablet_map.go [hunk 2]
```diff
@@ -384,3 +371,257 @@ func (itc *internalTabletConn) StreamHealth(ctx context.Context) (<-chan *queryp
...
 }
+
+//
+// TabletManagerClient implementation
+//
+
+// internalTabletManagerClient implements tmclient.TabletManagerClient
+type internalTabletManagerClient struct{}
+
+func (itmc *internalTabletManagerClient) Ping(ctx context.Context, tablet *topo.TabletInfo) error {
```
Comment (michael-berlin) [inline go/cmd/vtcombo/tablet_map.go]: nit: One letter for the receiver would have been enough e.g. just "t"?

File: test/vttest_sample_test.py [hunk 3]
```diff
@@ -73,19 +73,15 @@ def test_standalone(self):
...
-    utils.pause('good time to test vtcombo with database running')
-
+    # build the vtcombo address and protocol
     protocol = protocols_flavor().vttest_protocol()
     if protocol == 'grpc':
       vtagte_addr = 'localhost:%d' % config['grpc_port']
     else:
       vtagte_addr = 'localhost:%d' % config['port']
     conn_timeout = 30.0
+    utils.pause('good time to test vtcombo with database running on'
```
Comment (michael-berlin) [inline test/vttest_sample_test.py]: Instead "good time" I would prefer to be here more direct:

```
Paused test after vtcombo was started.

For manual testing, connect to vtgate at: %s using protocol: %s.

Press enter to continue.
```

File: test/vttest_sample_test.py [hunk 4]
```diff
@@ -145,6 +141,43 @@ def test_standalone(self):
...
+         '-server', vtagte_addr,
+         '-stderrthreshold', 'INFO',
+         'ListAllTablets', 'test',
+        ], trap_output=True)
+    num_master = 0
+    num_replica = 0
+    num_rdonly = 0
+    num_dash_80 = 0
+    num_80_dash = 0
+    for line in out.splitlines():
```
Comment (michael-berlin) [inline test/vttest_sample_test.py]: We don't have such a code already as a method?

**Comment by michael-berlin** [comment]
LGTM

File: go/cmd/vtcombo/tablet_map.go (hunk 2 above)
Comment (alainjobart) [inline go/cmd/vtcombo/tablet_map.go]: already have a different class in the previous section, I wanted the receiver to be different too, and easy to search for.

File: test/vttest_sample_test.py (hunk 4 above)
Comment (alainjobart) [inline test/vttest_sample_test.py]: we do but it's in the vtctld client code, which this one is not. making it generic is too big of a hammer I think.

File: test/vttest_sample_test.py (hunk 3 above)
Comment (alainjobart) [inline test/vttest_sample_test.py]: Done.

File: go/cmd/vtcombo/tablet_map.go (hunk 1 above)
Comment (alainjobart) [inline go/cmd/vtcombo/tablet_map.go]: done.

**Pull request merged**